webapp/src/shared/components/Input/Input.css

```
@@ -0,0 +1,50 @@
+.input {
```

just in case it gets override for some reason, what about defining default color as white? (right now labels are white because they inherit the color from root)

```suggestion
.input {
  color: var(--color-light); 
```

---

Yes, good point. Done